snuba/web/db_query.py

```diff
@@ -497,6 +538,8 @@ def record_cache_hit_type(hit_type: int) -> None:
         if hit_type == RESULT_VALUE:
             stats["cache_hit"] = 1
             span_tag = "cache_hit"
+            if sorted_key_exists:
+                metrics.increment("sorted_and_unsorted_cache_hit")
```

Comment:
You need to also track the other cases of this experiment. There should be 4:

**Sorted/Unsorted**
Miss/Miss
Miss/Hit
Hit/Miss <- the one we care about most
Hit/Hit